opengemini-client-jdk/src/test/java/io/opengemini/client/jdk/OpenGeminiJdkClientTest.java

```diff
@@ -194,4 +197,69 @@ void testWriteBatchPoints() {
         Assertions.assertTrue(x.getColumns().contains("field0"));
         Assertions.assertTrue(x.getColumns().contains("tag0"));
     }
+
+    @SneakyThrows
+    @Test
+    void testRetentionPolicyNormal() {
+        String testRpName = "testRpName";
+        ArrayList<RpConfig> rps = new ArrayList<>();
+        rps.add(new RpConfig(testRpName + 0, "3d", "", ""));
+        rps.add(new RpConfig(testRpName + 1, "3d", "1h", ""));
+        rps.add(new RpConfig(testRpName + 2, "3d", "1h", "7h"));
+        rps.add(new RpConfig(testRpName + 3, "365d", "", ""));
+
+        String database = "testRpDatabase0001";
+        CompletableFuture<Void> createdb = openGeminiJdkClient.createDatabase(database);
+        createdb.get();
+
+        for (int i = 0; i < rps.size(); i++) {
+            Boolean isDefaultRp = Boolean.FALSE;
+            if (i == 4) {
+                isDefaultRp = Boolean.TRUE;
+            }
+            CompletableFuture<Void> createRsp = openGeminiJdkClient.createRetentionPolicy(
+                    database, rps.get(i), isDefaultRp);
+            createRsp.get();
+            Thread.sleep(2000);
```

Comment:
I think in java we can use `Awaitility` instead of sleep?

Reply:
hope to keep the dependencies of this Java project simple